app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<section class="container flex food-placeholder">
+<div class="food flex">
+  <%= link_to new_recipe_path do %>
+    <button>Add Recipe</button>
+  <% end %>
+
+  <% @recipes.each do |recipe| %>
+  <div class='recipe flex'>
+    <div class="recipe-details flex">
+      <h2><%= link_to recipe.name, recipe_path(recipe) %></h2>
+      <%= button_to "Remove", recipe_path(recipe.id),data: { turbo_method: :delete, turbo_confirm: "Are you sure?"}, method: :delete, class: "delete" %>
```

Comment:
- [ ] Kindly make sure that the user can remove a recipe only when he was the one to create it as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=If%20the%20user%20is%20the%20owner%20of%20the%20recipe%2C%20should%20allow%20the%20user%20to%20delete%20it.).